samples/net/sockets/echo_service/src/main.c

```diff
@@ -25,13 +25,13 @@ LOG_MODULE_REGISTER(net_echo_server_svc_sample, LOG_LEVEL_DBG);
 BUILD_ASSERT(MAX_SERVICES > 0, "Need at least 4 poll fds, increase CONFIG_ZVFS_POLL_MAX");
 
 K_MUTEX_DEFINE(lock);
-static struct pollfd sockfd_udp[1] = {
-	[0] = { .fd = -1 }, /* UDP socket */
-};
 static struct pollfd sockfd_tcp[MAX_SERVICES];
+static int tcp_socket = -1;
+static int udp_socket = -1;
```

Comment:
The commit makes no mention of UDP changes, please make them in a separate commit or update the commit description.

Reply:
I was referring to a commit, which from the description claimed to be moving accepting of the TCP connections into the socket service, yet it had unrelated changes to UDP service part of the sample. Just asing to be explicit when describing commits.